spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -176,7 +176,7 @@ object SpdxLicenseAliasMapping {
     /**
      * The map of varied SPDX license ids associated with their corresponding SPDX expression.
      */
-    val mapping =
+    internal val mapping =
```

Comment:
I believe this doesn't work as we access it from `DeclaredLicenseProcessorTest`. But if it worked, we should do the same in `SpdxDeclaredLicenseMapping`, too.

Reply:
Indeed, but I don't like making things public only to access them from tests, I will check if there is a simple fix or drop the commit.

Reply:
I have dropped the commit because I do not want to spend more time on this right now.